Building pre-release 1.4.2 fails when not on a git snapshot

- Delta Chat Version: 1.4.2
- Expected behavior: npm run build succeeds
- Actual behavior: Build fails on `build:shared:version` step, as git describe in [bin/build-shared-version-info.js](https://github.com/deltachat/deltachat-desktop/blob/master/bin/build-shared-version-info.js#L6) doesn't work when building from release tar-ball
- Steps to reproduce the problem: 

I use docker here to get a clean build env, but this doesn't affect the error.

    $ docker run --rm -ti --entrypoint=bash node
    # curl -LO https://github.com/deltachat/deltachat-desktop/archive/v1.4.2.tar.gz
    # tar xzf v1.4.2.tar.gz && cd deltachat-desktop-1.4.2/
    # npm install
    [...] works fine
    # npm run build
    [...]
    > deltachat-desktop@1.4.2 build:shared:version /deltachat-desktop-1.4.2
    > node ./bin/build-shared-version-info.js
    
    <Buffer 66 61 74 61 6c 3a 20 4e 6f 74 20 61 20 67 69 74 20 72 65 70 6f 73 69 74 6f 72 79 20 28 6f 72 20 61 6e 79 20 6f 66 20 74 68 65 20 70 61 72 65 6e 74 20 ... 19 more bytes>
    (node:240) UnhandledPromiseRejectionWarning: Error: getting git commit failed
        at gatherBuildInfo (/deltachat-desktop-1.4.2/bin/build-shared-version-info.js:9:11)
        at Object.<anonymous> (/deltachat-desktop-1.4.2/bin/build-shared-version-info.js:21:1)
        at Module._compile (internal/modules/cjs/loader.js:1200:30)
        at Object.Module._extensions..js (internal/modules/cjs/loader.js:1220:10)
        at Module.load (internal/modules/cjs/loader.js:1049:32)
        at Function.Module._load (internal/modules/cjs/loader.js:937:14)
        at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
        at internal/main/run_main_module.js:17:47
    (Use `node --trace-warnings ...` to show where the warning was created)
    (node:240) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
    (node:240) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
    
    > deltachat-desktop@1.4.2 build:backend /deltachat-desktop-1.4.2
    > tsc --build src/main
    
    src/main/deltachat/controller.ts:24:42 - error TS2307: Cannot find module '../../shared/build-info'.
    
    24 import { VERSION, BUILD_TIMESTAMP } from '../../shared/build-info'
                                                ~~~~~~~~~~~~~~~~~~~~~~~~~
    
    Found 1 error.

Building succeeds by hard-coding `VERSION`, `BUILD_TIMESTAMP` and `GIT_REF`, but that's obviously not optimal :)

--- a/bin/build-shared-version-info.js
+++ b/bin/build-shared-version-info.js
@@ -4,15 +4,17 @@ const { join } = require('path')
 
 async function gatherBuildInfo() {
   const { status, stdout, stderr } = spawnSync('git', ['describe'])
+  let git_ref = "None"
   if (status !== 0) {
     console.log(stderr)
-    throw new Error('getting git commit failed')
+  } else {
+    git_ref = stdout.toString().replace(/\n/g, '')
   }
   const package = await fs.readJSON(join(__dirname, '../package.json'))
   return {
     VERSION: package.version,
     BUILD_TIMESTAMP: Date.now(),
-    GIT_REF: stdout.toString().replace(/\n/g, ''),
+    GIT_REF: git_ref
   }
 }
 

